resources/lang/en/auth.php

```diff
@@ -73,7 +73,8 @@
     'confirmation_title' => 'Verify Your Email Address',
     'confirmation_fresh' => 'A fresh verification link has been sent to your email address.',
     'confirmation_check' => 'Before proceeding, please check your email for a verification link.',
-    'confirmation_request_another' => 'If you did not receive the email <a href=":url">click here to request another</a>.',
+    'confirmation_request_another' => 'If you did not receive the email',
```

Comment:
I would rather write
```suggestion
    'confirmation_request_another' => 'If you did not receive the email <button type="submit" class=":class">click here to request another</button>.',
```

and use this single text in the form